src/azul/service/hca_response_v5.py

```diff
@@ -165,6 +165,7 @@ class SummaryRepresentation(JsonObject):
     donorCount = IntegerProperty()
     labCount = IntegerProperty()
     totalCellCount = FloatProperty()
+    projectEstimatedCellCount = FloatProperty()
```

Comment:
https://github.com/DataBiosphere/azul/pull/3361#discussion_r696207701

Reply:
If there's an easy fix, then this and the property above (totalCellCount) should be changed.

Is it a float in ES to allow for astronomical values?

Reply:
> Is it a float in ES to allow for astronomical values?

ElasticSearch uses Kahan summation for "better accuracy than naive summation" https://github.com/elastic/elasticsearch/issues/27807 .

Reply:
The fact that it is a float predates that ticket, so the ticket does not sufficiently motivate why ES uses floats for sums.

Reply:
This thread https://github.com/elastic/elasticsearch/issues/9545#issuecomment-81832865 from 2015 discusses the usage doubles for all metrics aggregations, and the problems preventing them from changing the behavior. 

> Not only the interface is an issue but also the internal implementation: for simplicity we handle everything as doubles. We could potentially fix the min and max aggs, but I don't think eg. the sum aggregation is fixable since you can easily overflow a long while a double gracefully loses accuracy as the sum increases.

> I would like to fix it, but it is not straightforward and also looks impossible to do without breaking all users of the Java transport client. So I am tempted to close it and document the behaviour. Maybe we can reconsider when all clients talk to Elasticsearch using the REST layer.